{{cookiecutter.project_slug}}/compose/production/caddy/Caddyfile

```diff
@@ -1,5 +1,5 @@
 www.{% raw %}{$DOMAIN_NAME}{% endraw %} {
-    redir https://{{cookiecutter.domain_name}}
+    redir https://{% raw %}{$DOMAIN_NAME}{% endraw %} {
```

Comment:
It looks like there is an extra `{` at the end of the line. Assume that was a wrong copy/paste? 

Not sure why we didn't use the environment variable here, the new version seems dryer to me 👍

Reply:
yep, sorry about that.  i fixed it and merged in the latest changes to master.